Source/WebKit/UIProcess/Cocoa/GPUProcessProxyCocoa.mm

```diff
@@ -89,6 +89,19 @@
     return SandboxExtension::createHandlesForMachLookup({ "com.apple.gputools.service"_s, }, std::nullopt);
 }
 
+#if USE(EXTENSIONKIT)
+void GPUProcessProxy::sendBookmarkDataForCacheDirectory()
+{
+    NSError *error = nil;
+    NSURL *directoryURL = [[NSFileManager defaultManager] URLForDirectory:NSLibraryDirectory inDomain:NSUserDomainMask appropriateForURL:nil create:NO error:&error];
+    auto url = adoptNS([[NSURL alloc] initFileURLWithPath:@"Caches/com.apple.WebKit.GPU/" relativeToURL:directoryURL]);
+    error = nil;
+    NSData* bookmark = [url bookmarkDataWithOptions:NSURLBookmarkCreationMinimalBookmark includingResourceValuesForKeys:nil relativeToURL:nil error:&error];
```

Comment:
For the record, this patch introduced file system work on the main thread of the UIProcess, which is never OK. It regressed app launch as a result.